docs/4.x/request-parameters.md

```diff
@@ -0,0 +1,75 @@
+---
+category: Develop
+title: Handling Request Parameters
+---
+# Handling Request Parameters
+
+If you need to access a query or post parameter value, use the [`Common::getRequestVar($varName, $varDefault = null, $varType = null, $requestArrayToUse = null)`](/api-reference/Piwik/Common#getrequestvar) method.
+
+**To avoid XSS vulnerabilities, never access `$_GET`/`$_POST` directly, always go through [Common::getRequestVar](/api-reference/Piwik/Common#getrequestvar).**
+
```

Comment:
```suggestion

Using `Common::getRequestVar` will automatically sanitize the request variables using [Common#sanitizeInputValues](/api-reference/Piwik/Common#sanitizeinputvalues). For arrays and parsed json data this is done recursively. Internally this will escape e.g. special html characters. If you need to work with unescaped values, you may need to apply [Common#unsanitizeInputValues](/api-reference/Piwik/Common#unsanitizeinputvalues).

```

Reply:
@sgiehl sorry I thought that was my PR, commit suggestion sorry about that .